docs/status.md

```diff
@@ -52,7 +52,7 @@ one of the following tracking labels.
 | atan2                    | no            | yes*         | yes*           | yes             | no          |
 | batch_norm_grad          | no            | yes*         | yes*           | no              | no          |
 | batch_norm_inference     | yes           | revisit      | yes            | no              | no          |
-| batch_norm_training      | no            | yes*         | yes*           | no              | no          |
+| batch_norm_training      | no            | revisit      | yes            | no              | no          |
 | bitcast_convert          | no            | yes*         | infeasible     | yes             | no          |
```

Comment:
marked revisit like in `batch_norm_inference`

Reply:
@burmako

Reply:
#369

Reply:
Nice catch! Indeed, this should be `revisit`.